pkg/proxy/userspace/proxier.go

```diff
@@ -231,7 +234,11 @@ func NewCustomProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptab
 
 	err = setRLimit(64 * 1000)
 	if err != nil {
-		return nil, fmt.Errorf("failed to set open file handler limit: %v", err)
+		if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.KubeletInUserNamespace) && libcontainersystem.RunningInUserNS() {
+			klog.ErrorS(err, "failed to set open file handler limit to 64000 (running in UserNS, ignoring)")
+		} else {
+			return nil, fmt.Errorf("failed to set open file handler limit to 64000: %v", err)
```

Comment:
I think the new code should use `%w` for error wrapping.

Reply:
(this is not a new code, but maybe makes sense to fix it while moving it)

Reply:
updated